src/Sarif.Sarifer/BackgroundAnalyzerBase.cs

```diff
@@ -73,7 +77,7 @@ public async Task AnalyzeAsync(string path, string text, CancellationToken cance
 
                 await writer.FlushAsync().ConfigureAwait(continueOnCapturedContext: false);
 
-                await this.WriteToSinksAsync(path, stream).ConfigureAwait(false);
+                await this.WriteToSinksAsync(path, stream, false).ConfigureAwait(false);
```

Comment:
>false [](start = 59, length = 5)

Use named parameter so it's clear what the "false" means: `cleanAll: false`.

I suggest the same for all calls to `ConfigureAwait`: `continueOnCapturedContext: false`, even though it's verbose.